windows/src/global/delphi/general/Upload_Settings.pas

```diff
@@ -50,6 +49,7 @@ interface
   URLPath_ArchivedDownloads = '/go/desktop/'+SKeymanVersion+'/archived-downloads';
   URLPath_Support = '/go/'+SKeymanVersion+'/support';
   URLPath_Privacy = '/go/'+SKeymanVersion+'/privacy';
+  URLPath_Community = '/go/'+SKeymanVersion+'/community';
```

Comment:
Don't the 3 links above also need to be `'/go/desktop'+SKeymanVersion+...`?
The go/desktop/web.config has
```
                <rule name="/go/desktop/X.Y/support" stopProcessing="true">
                  <match url="^([1-9][0-9]\.[0-9])/support" />
                  <action type="Redirect" url="/support" />
                </rule>

                <rule name="/go/desktop/X.Y/privacy" stopProcessing="true">
                  <match url="^([1-9][0-9]\.[0-9])/privacy" />
                  <action type="Redirect" url="/privacy" />
                </rule>
```

Reply:
I figured these should be global redirects given they have no app-specific detail. See the new PR at keymanapp/keyman.com#93.